### ISSUE TITLE
test_imap uses deprecated Deferred.setTimeout

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#2051]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)| @exarkun reported|
|-|-|
|Trac ID|trac#2051|
|Type|enhancement|
|Created|2006-09-02 17:59:13Z|

<details><summary>Searchable metadata</summary>

```
trac-id__2051 2051
type__enhancement enhancement
reporter__exarkun exarkun
priority__highest highest
milestone__ 
branch__ 
branch_author__ 
status__closed closed
resolution__fixed fixed
component__mail mail
keywords__ 
time__1157219953000000 1157219953000000
changetime__1157565504000000 1157565504000000
version__None None
owner__ 

```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)<a name="note_1"></a>|@exarkun removed owner|
|-|-|

The call is basically pointless.  Probably left over from some debugging session before trial supported test methods which returned Deferreds.  Here is a patch which removes it:

```python
Index: twisted/mail/test/test_imap.py
===================================================================
--- twisted/mail/test/test_imap.py      (revision 18024)
+++ twisted/mail/test/test_imap.py      (working copy)
@@ -1393,7 +1393,6 @@
         d1 = self.connected.addCallback(strip(login))
         d1.addCallbacks(strip(append), self._ebGeneral)
         d1.addCallbacks(self._cbStopClient, self._ebGeneral)
-        d1.setTimeout(5)
         d2 = self.loopback()
         d = defer.gatherResults([d1, d2])
         return d.addCallback(self._cbTestPartialAppend, infile)
```

**Comment by twisted-trac** [comment]
|[<img alt="jerub's avatar" src="https://avatars.githubusercontent.com/u/315461?s=50" width="50" height="50">](https://github.com/jerub)<a name="note_2"></a>|@jerub set owner to @exarkun|
|-|-|

Thankyou, please merge.

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)<a name="note_3"></a>|@exarkun set status to `closed`|
|-|-|

(In [18061]) Apply patch from [[#2051](https://github.com/twisted/twisted/issues/2051)](https://github.com/twisted/twisted/issues/2051) - remove pointless Deferred.setTimeout call

Author: exarkun
Reviewer: jerub
Fixes #2051

For no particular reason, an IMAP4 test case was giving a Deferred a five
second timeout using the deprecated Deferred.setTimeout API.  Since there
was no good reason for this, just remove it.

**Comment by twisted-trac** [comment]
|<img alt="Automation's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50">|Automation removed owner|
|-|-|
